data/DC/incentives.json

```diff
@@ -0,0 +1,330 @@
+      "en": "The affordable home electrification program can help you make the switch from natural gas and electrify your home heating and water heating systems."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-16",
+    "authority_type": "city",
+    "authority": "dc-dc-sustainable-energy-utility",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "heat_pump_water_heater",
+    "program": "dc_dCSustainableEnergyUtility_affordableHomeElectrificationProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 6000
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "The affordable home electrification program can help you make the switch from natural gas and electrify your home heating and water heating systems."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-19",
+    "authority_type": "city",
+    "authority": "dc-dc-sustainable-energy-utility",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "electric_panel",
+    "program": "dc_dCSustainableEnergyUtility_affordableHomeElectrificationProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 2300
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "The affordable home electrification program can help you make the switch from natural gas and electrify your home heating and water heating systems."
+    },
+    "start_date": 2023,
+    "end_date": 2024
+  },
+  {
+    "id": "DC-21",
+    "authority_type": "state",
+    "authority": "dc-dc-department-of-energy-and-environment",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "weatherization",
+    "program": "dc_dCDepartmentOfEnergyAndEnvironment_weatherizationAssistanceProgram",
+    "amount": {
+      "type": "percent",
+      "number": 1
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Provides low-income residents with technical and financial assistance to help reduce their energy bills by making their homes more energy efficient."
+    },
+    "start_date": 2023,
+    "end_date": 2024,
+    "bonus_available": true,
+    "low_income": "dc-dc-department-of-energy-and-environment-60-ami"
+  },
+  {
+    "id": "DC-24",
+    "authority_type": "city",
+    "authority": "dc-dc-sustainable-energy-utility",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "rooftop_solar_installation",
+    "program": "dc_dCSustainableEnergyUtility_solarForAllSingle-FamilyProgram",
+    "amount": {
+      "type": "percent",
+      "number": 1
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Local solar contractors design and install solar photovoltaic (PV) systems at no cost to income-qualified homeowners."
```

Comment:
This largely has the right info but should be framed for the customer perspective: e.g. 100% of costs covered for income-qualified homeowners to have solar photovoltaic (PV) systems installed by local contractors.